module/renderer/src/commonMain/kotlin/org/cru/godtools/shared/renderer/content/RenderLink.kt

```diff
@@ -0,0 +1,39 @@
+package org.cru.godtools.shared.renderer.content
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.heightIn
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import org.cru.godtools.shared.renderer.content.extensions.handleClickable
+import org.cru.godtools.shared.renderer.state.State
+import org.cru.godtools.shared.tool.parser.model.Link
+
+private val Link_Minimum_Height = 48.dp
+
+@Composable
+fun ColumnScope.RenderLink(link: Link, state: State, modifier: Modifier = Modifier) {
+    val scope = rememberCoroutineScope()
+
+    Box(
+        modifier = Modifier
+            .clickable { link.handleClickable(state, scope) }
+            .fillMaxWidth()
+            .heightIn(min = Link_Minimum_Height),
+        contentAlignment = Alignment.Center
```

Comment:
instead of nesting the text in a Box, try this set of modifiers directly on `RenderTextNode`:

```
    .fillMaxWidth()
    .clickable {}
    .heightIn(min = Link_Minimum_Height)
    .wrapContentHeight(Alignment.CenterVertically)
```

Reply:
ooo, there is a `Modifier.minimumInteractiveComponentSize()` you can use instead of hardcoding our own `heightIn()` with a minimum touch size. it also centers the content for you.

```
    .minimumInteractiveComponentSize()
    .clickable {}
```

Reply:
Hey @frett ```Modifier.minimumInteractiveComponentSize()``` also worked nicely for sizing and centering the content vertically.
The only issue I had was with tap area.  If I don't wrap the RenderTextNode in a Box element, no matter what I do with the modifiers the tap area will be tappable in the white space to the left and right of the text because of the fillMaxWidth on the Text.  Changing the order of the clickable effected how the tap highlighted and not the tap itself.  I have a sample project I can show you what I was seeing, but, let me know if there's another way to have the tap only register on the text without wrapping in a Box element.